GVRf/Framework/src/org/gearvrf/animation/keyframe/GVRAnimationController.java

```diff
@@ -0,0 +1,36 @@
+package org.gearvrf.animation.keyframe;
+
+public abstract class GVRAnimationController {
+    private static final float TOL = 1e-6f;
+    protected GVRKeyFrameAnimation animation;
```

Comment:
I would make this final unless you plan to implement a setKeyFrameAnimation method.